server/middleware/features-cohort.js

```diff
@@ -0,0 +1,6 @@
+export function determineFeaturesCohort(config) {
+  return (ctx, next) => {
+    config.featuresCohort = ctx.cookies.get('WC_featuresCohort') || 'default';
```

Comment:
This is against the norm - why not attach it to the context - mutating config makes this a little magic.